### PR TITLE
BSP-266 Multiple owners

### PR DESCRIPTION
**Pull request by podefr**
- Allow contract to have multiple owners
- Owners can be added/removed as needed

### CODE REVIEW

File: misc/estimate_batch.test.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+const { TezosToolkit, MichelsonMap, OpKind } = require("@taquito/taquito");
```
Comment (podefr) [inline misc/estimate_batch.test.js:1]: the files in the `misc` folder are just a bunch of scripts I use for testing various aspects of the contracts. This is not production code, just test snippets

File: misc/estimate_contract_call.test.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+const { TezosToolkit, MichelsonMap } = require("@taquito/taquito");
```
Comment (podefr) [inline misc/estimate_contract_call.test.js:1]: the files in the `misc` folder are just a bunch of scripts I use for testing various aspects of the contracts. This is not production code, just test snippets

File: misc/estimate_cost.test.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ async function originateWarehouse() {
     const originationOp = await Tezos.contract.originate({
```
Comment (podefr) [inline misc/estimate_cost.test.js:16]: the files in the `misc` folder are just a bunch of scripts I use for testing various aspects of the contracts. This is not production code, just test snippets

File: misc/operation_confirmation.js [hunk 4]
```diff
@@ -0,0 +1,197 @@
+const { TezosToolkit, MichelsonMap } = require("@taquito/taquito");
```
Comment (podefr) [inline misc/operation_confirmation.js:1]: the files in the `misc` folder are just a bunch of scripts I use for testing various aspects of the contracts. This is not production code, just test snippets

**Review by 0x103 — APPROVED** [review]

**Pull request merged**